openai/openai.go

```diff
@@ -654,8 +639,8 @@ func (w *ChatWriter) writeResponse(data []byte) (int, error) {
 	// chat chunk
 	if w.stream {
 		c := toChunk(w.id, chatResponse)
-		if w.streamUsage {
-			c.Usage = &nullChunkUsage
+		if w.streamOptions != nil && w.streamOptions.IncludeUsage {
+			c.Usage = &Usage{}
 		}
```

Comment:
```suggestion
```

Reply:
Looks like I don't have permission to modify the PR directly. No worries, I'll just bring in the change. Thanks for the cleanup PR!